src/requests/session.ts

```diff
@@ -104,6 +105,44 @@ export async function getDetailedSessionById(
   return response.lowcal_sessions_by_pk;
 }
 
+export async function getSessionBreadcrumbs(
+  client: GraphQLClient,
+  sessionId: string
+): Promise<Breadcrumbs | null> {
+  const response: Record<
+    "lowcal_sessions_by_pk",
+    { breadcrumbs: Breadcrumbs }
+  > = await client.request(
+    gql`
+      query GetSessionById($id: uuid!) {
+        lowcal_sessions_by_pk(id: $id) {
+          breadcrumbs: data(path: "breadcrumbs")
+        }
+      }
+    `,
+    { id: sessionId }
+  );
+  return response?.lowcal_sessions_by_pk?.breadcrumbs || null;
+}
```

Comment:
This function was used in earlier versions of this refactor but not anymore. I thought it was probably worth keeping for future reuse.